bin/common/utils.js

```diff
@@ -43,17 +43,24 @@ exports.loadNuxtConfig = (argv) => {
   options.mode =
     (argv.spa && 'spa') || (argv.universal && 'universal') || options.mode
 
-  return options
-}
+  // Server options
+  if (!options.server) {
```

Comment:
Instead of checking if `options.server` is set, better add `{ host: undefined, port: undefined }` as defaults in the [nuxt.config.js](https://github.com/nuxt/nuxt.js/blob/dev/lib/common/nuxt.config.js)

Reply:
Is this one necessary ?